app/models/report.rb

```diff
@@ -1,6 +1,8 @@
 require_dependency 'topic_subtype'
 
 class Report
+  extend ActionView::Helpers::NumberHelper
```

Comment:
I am not very happy about extending the model class with a helper, so I am open to suggestions 😃

Reply:
A simple solution would be to give this report its own file, we already do this for some reports. This way we would only include it for this specific report and not all reports.

Reply:
I think this is a good approach. Looking at the `report.rb`, I could imagine, even from a directory structure, it would be better having a `lib/reporting/...` directory where each report type perhaps have its own class.

I am happy to go by your suggestion in this PR, or leave the refactor to a subsequent one. What do you think?

Reply:
I think you can change this one to work like this, no need to change everything yet. I have let multiple refactorings on the side waiting for old dashboard to be removed. It would have forced me to have duplicated logic in some cases, once we don’t have the old one we can go aggressive on refactoring without fearing of breaking old features.

Also I think `lib/reports/` might be slightly better`

Reply:
> I think you can change this one to work like this, no need to change everything yet.

That makes sense 👍